doc/topics/pillar/index.rst

```diff
@@ -236,6 +236,51 @@ The resulting pillar will be as follows:
     For example, in the above scenario conflicting key values in ``services``
     will overwrite those in ``packages`` because it's at the bottom of the list.
 
+
+Referencing Other Pillars
+=========================
+
+.. versionadded:: ?????
```

Comment:
Now that we're past the Carbon release cycle, this can be changed to ``.. versionadded:: Nitrogen``